recipes/darmonpoints/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "darmonpoints" %}
+{% set version = "7.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b827ad80527d3356bef7b7c2480d73e7a060a8f4c183729990de0b166625c6bf
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython
+    - python
+    - pip
+  run:
+    - python
+    - fplll
+    - fpylll
+    - gap-defaults
+    - matplotlib-base
+    - sagelib
+    - sagemath-db-conway-polynomials
+    - sagemath-db-elliptic-curves
+    - zn_poly
+
+test:
+  imports:
+    - darmonpoints.mixed_extension.get_word_rep_fast
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: mmasdeu.github.io/darmonpoints
+  summary: 'A package to compute with Darmon points'
+  description: |
+    The darmonpoints package can compute many different types of what is known as Darmon points.
```

Comment:
Maybe mention what field this is in, mathematics or more specifically number theory  (?)